### PR TITLE
ISSUE-64: Anonymize/pseudonymize a sentence

### PR DESCRIPTION
**Pull request by alimuhammadahmer**
This is a pull request to pseudonymise a particular sentence. It forwards the request to the Pseudonymise Service and returns the response.

### CODE REVIEW

File: DSL.Ruuter/POST/pseudonymise.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+pseudonymise:
+  call: http.post
+  args:
+    url: http://web:5001/predict
```
Comment (turnerrainer) [inline DSL.Ruuter/POST/pseudonymise.yml:4]: What endpoint is `http://web:5001/predict`?

File: README.md [hunk 2]
```diff
@@ -11,5 +11,14 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/insert_regex.s
 
 https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/regex.yml
 
+```
+```
```
Comment (turnerrainer) [inline README.md:15]: Please add a sample curl request.

File: README.md [hunk 3]
```diff
@@ -11,5 +11,14 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/insert_regex.s
...
+``` 
+
+### Anonymize/pseudonymize a sentence
+
+https://github.com/buerokratt/Data-Anonymizer/issues/64
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/pseudonymise.yml
+
 ```
 ```
```
Comment (turnerrainer) [inline README.md:24]: Please add a sample curl request.

**Review by turnerrainer — CHANGES_REQUESTED** [review]

File: DSL.Ruuter/POST/pseudonymise.yml (hunk 1 above)
Comment (alimuhammadahmer) [reply DSL.Ruuter/POST/pseudonymise.yml:4]: This is the endpoint which is hosted on our Anonymiser Service. It pseudonymises a particular text using the trained model.

File: README.md (hunk 2 above)
Comment (alimuhammadahmer) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (alimuhammadahmer) [reply README.md]: Done

**Review by turnerrainer — APPROVED** [review]

**Pull request merged**